tools/pythonpkg/tests/fast/api/test_read_csv.py

```diff
@@ -0,0 +1,105 @@
+import numpy
+import datetime
+import pandas
+import pytest
+import duckdb
+
+def TestFile(name):
+	import os
+	filename = os.path.join(os.path.dirname(os.path.realpath(__file__)),'..','data',name)
+	return filename
+
+class TestReadCSV(object):
```

Comment:
Can we also add a test with the parallel_csv_reader?

Reply:
> So maybe we should just upgrade it to a csv reader option?

I think that is a good idea

Reply:
Removing the DBConfig setting as well, or not yet?